eth_account/account.py

```diff
@@ -187,7 +187,7 @@ def encrypt(private_key, password, kdf=None):
             key_bytes = HexBytes(private_key)
 
         if kdf is None:
-            kdf = __class__.default_kdf  # noqa: F821
+            kdf = cls.default_kdf  # noqa: F821
```

Comment:
```suggestion
            kdf = cls.default_kdf
```

Should be able to drop the flake8 annotation now

Reply:
Removed 👍 

https://github.com/ethereum/eth-account/pull/41/commits/690bbdd0c59c5bc567fd03cbc0b06a0b677424b5#diff-a9c6ca5d3c0ddc6c626bcee1142503c9L190